### ISSUE TITLE
Initial notes for WL

### ISSUE THREAD

**Issue by rmandelb** [wadawson/Enabling-LSST-Dark-Energy-Science#2]
Hi all,

Will and I have been e-mailing but decided to move this onto GitHub so that others could have the option of following along / commenting if they want.

For context, I put a sketchy set of notes on the WL case in the document earlier this week.  They are actually longer than what Jeff requested, but I tried to be clear and thorough so that Will and I could iterate and make sure we agree what to say, and then we can condense to something briefer before the telecon Monday afternoon.  (Also, this way the longer reasoning exists in the git history in case we need to reconstruct it later.)

To summarize:
- In https://github.com/wadawson/Enabling-LSST-Dark-Energy-Science/commit/109c7c9d967a61a532418491316cabf6e92db792 , I put in words concluding that we are unlikely to need to talk about imaging for galaxy morphologies after all.  (I had mentioned it on the telecon, but when I thought it through carefully I realized it's not necessary.)
- In https://github.com/wadawson/Enabling-LSST-Dark-Energy-Science/commit/f8d54ca7304f9e75fb7b6fe19ab694365d22d15a , I put the beginning of an intrinsic alignments discussion but with open questions.

**Comment by rmandelb** [comment]
In https://github.com/wadawson/Enabling-LSST-Dark-Energy-Science/commit/4b4120ec02c6a5a44b6f74790fd0456f3db8618b , Will added another possible WL science case.  Copying from his e-mail (with permission): "I’ve briefly tried to think of how much added morphological information there might be in obtaining spectra of a subsample of the LSST galaxies and I can’t come up with anything compelling (other than constraints that are better covered under the photo-z purview). The best thing I could think of is Tully-Fisher lensing (e.g. Huff et al. 2013), although I don’t know how complementary it is to LSST. It might be work pinging Eric at some point."

My reply was "I’ll ask Eric for an update, but my feeling at this point is that we are largely supposed to think of ways the US OIR system can support the LSST DE science case, and I’m not sure we have to worry about enabling a speculative alternate WL method (as opposed to say getting spec-z for photo-z calibration for the main WL science case)."  So i did ask Eric if there is a proof of concept for this method, but I think this science case is still likely to rank lower than spec-z for photo-z calibration or IA priors.

**Comment by rmandelb** [comment]
To summarize where we are now:
- For the galaxy morphology question, we seem to be largely in agreement that standard WL methods don't require a mention in this report.
- We are looking into the kinematic lensing method, but in the absence of a proof of concept or a strong reason to favor it over supporting the standard lensing method, it seems this may be lower priority and thus not need to be mentioned.
- We haven't discussed the IA stuff yet.

**Comment by janewman-pitt-edu** [comment]
Yeah, our focus is supposed to be things that 'maximize the science enabled
by LSST' -- I don't think T-F lensing really fits that bill (LSST-depth
imaging is almost certainly overkill for that).

Best,

Jeff

On Thu, Mar 24, 2016 at 9:09 PM, Rachel Mandelbaum <notifications@github.com

> wrote:
> 
> In 4b4120e
> https://github.com/wadawson/Enabling-LSST-Dark-Energy-Science/commit/4b4120ec02c6a5a44b6f74790fd0456f3db8618b
> , Will added another possible WL science case. Copying from his e-mail
> (with permission): "I’ve briefly tried to think of how much added
> morphological information there might be in obtaining spectra of a
> subsample of the LSST galaxies and I can’t come up with anything compelling
> (other than constraints that are better covered under the photo-z purview).
> The best thing I could think of is Tully-Fisher lensing (e.g. Huff et al.
> 2013), although I don’t know how complementary it is to LSST. It might be
> work pinging Eric at some point."
> 
> My reply was "I’ll ask Eric for an update, but my feeling at this point is
> that we are largely supposed to think of ways the US OIR system can support
> the LSST DE science case, and I’m not sure we have to worry about enabling
> a speculative alternate WL method (as opposed to say getting spec-z for
> photo-z calibration for the main WL science case)." So i did ask Eric if
> there is a proof of concept for this method, but I think this science case
> is still likely to rank lower than spec-z for photo-z calibration or IA
> priors.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/wadawson/Enabling-LSST-Dark-Energy-Science/issues/2#issuecomment-201091630

## 

---

Jeffrey Newman                 Assoc. Prof., U. Pittsburgh Dept. of Physics
& Astronomy
(412) 592-3853                 www.phyast.pitt.edu/~janewman

**Comment by wadawson** [comment]
I am afraid I don't have much to contribute to the IA section, I haven't kept up with field at anywhere near the level you have. Perhaps it is worth consulting with others.

The only thing I could think of that might add a new angle to what is already in the text, that might avoid the need for densely sampling sample a small region of the sky, is to implement some method like Aragon-Calvo et al. (2014, 2015). Although I am not sure if we can get enough redshifts over the redshift ranges where IA is important for the method to work. Perhaps worth consulting with Miguel.

**Comment by rmandelb** [comment]
IA: Yes, I will ping a couple of people.

You mean http://adsabs.harvard.edu/abs/2015MNRAS.454..463A ?
So this is related to this question that I put in the text: "Question: can we use cross-correlations? Perhaps we could use a more sparse sampling of the density field over a wider area, and constrain IA using cross-correlations with the spectroscopic galaxies instead of the auto-correlations of them (using a method like that from Blazek et al 2012 or Chisari et al 2014). How to define the needed dataset for this? Have to think about it."  The sparse sampling of the density field over a wider area that one would need to implement the methods in Blazek et al or Chisari et al is similar to what you'd need to carry out the method in Aragon-Calvo et al 2015, and also I would presume it's similar to what you'd need for cross-correlation analysis to infer N(z).

**Comment by rmandelb** [comment]
@wadawson - I just updated the doc to reflect where we seem to be headed based on e-mails with others.  Please check it out and make sure you are OK with it before the telecon (or we can chat more on here before the telecon).

**Comment by rmandelb** [comment]
@janewman-pitt-edu - on Monday morning, there is a DESC WL telecon.  Would it be appropriate to mention this whole project on there, mention which WL science case Will and I have been planning to include, and ask people if they think there is something major that we're missing?  Obviously Will and I may have to do some filtering (just like the filtering that we did to my own idea about morphologies) but I thought this is still potentially useful.  If you think it's appropriate, I'll ask Joe and Michael if there will be time for that.

**Comment by janewman-pitt-edu** [comment]
Hi Rachel,

I�d think this should be fine, but am CC�ing Joan and Beth in case I�m off-base.

Best,

Jeff

**Comment by janewman-pitt-edu** [comment]
Yes! Very good.

Beth

**Comment by rmandelb** [comment]
great!  Will talk to Joe and Michael.

**Comment by rmandelb** [comment]
I think we're well past the "initial notes" stage, what with there being a whole discussion in there.

Closing #2.
